pybliometrics/scopus/affiliation_retrieval.py

```diff
@@ -44,6 +44,14 @@ def date_created(self) -> Optional[Tuple[int, int, int]]:
     def document_count(self) -> int:
         """Number of documents for the affiliation."""
         return int(self._json['coredata']['document-count'])
+    
+    @property
+    def document_entitlement_status(self) -> Optional[str]:
+        """Returns the document entitlement status, i.e. tells if the requestor 
+        is entitled to the requested resource.
+        Note: Only works with `ENTITLED` view.
+        """
+        return chained_get(self._json, ['document-entitlement', 'status'])
```

Comment:
Are you sure this works? If I understand this correctly, `self._json` is not defined for this view.

Reply:
Yes, it works. The json for this view looks like this:
`
{'document-entitlement': {'message': 'Requestor is entitled to the requested resource',
  'url': 'http://prod-app-internal.elsapi.elsst.com/ElsApi/content/abstract/doi/10.1109/Multi-Temp.2019.8866947',
  'status': 'ENTITLED'}}
`
The `chained_get` returns `None` since it does not find `['document-entitlement', 'status']`